web/src/pages/new_account.rs

```diff
@@ -140,14 +137,48 @@ impl Component for NewAccount {
             <hr />
 
             <h2> { "Network" } </h2>
-            <span class=classes!("boxes") >
-                { for network.iter().map(|node| html! {
-                    <MatButton
-                        label=node.to_owned()
-                        unelevated=true
-                    />
-                }) }
-            </span>
+
+            <style> { "
+                .boxes {
+                    position: absolute;
+                    width: 20em; height: 10em;
+                    margin-left: 10em;
+                    z-index: -1;
+
+                    border: dotted var(--light-grey);
+                }
+
+                .boxes > * {
+                    position: absolute;
+                    margin: 0px;
+
+                    background-color: white;
+                    border: solid;
+                    padding: 1em 2em;
+                }
+                @media (prefers-color-scheme: dark) {
+                    .boxes > * {
+                      background-color: var(--dark-grey);
+                    }
+                }
+
+                .boxes > :nth-child(1) {
```

Comment:
Funny - I imagined a triangle with the base on the bottom - but this is also nice.